runtime/src/runtime/policy.rs

```diff
@@ -339,12 +347,19 @@ pub mod policy_constants {
     pub const WORKER_KEY_CHANGE_DELAY: ChainEpoch = CHAIN_FINALITY;
 
     /// Minimum number of epochs past the current epoch a sector may be set to expire.
-    pub const MIN_SECTOR_EXPIRATION: i64 = 180 * EPOCHS_IN_DAY;
+    pub const MIN_SECTOR_COMMITMENT: i64 = 180 * EPOCHS_IN_DAY;
 
     /// Maximum number of epochs past the current epoch a sector may be set to expire.
     /// The actual maximum extension will be the minimum of CurrEpoch + MaximumSectorExpirationExtension
     /// and sector.ActivationEpoch+sealProof.SectorMaximumLifetime()
-    pub const MAX_SECTOR_EXPIRATION_EXTENSION: i64 = 540 * EPOCHS_IN_DAY;
+    pub const MAX_SECTOR_COMMITMENT_EXTENSION: i64 = 540 * EPOCHS_IN_DAY;
+
+    /// Maximum number of epochs the proof can be valid since activation or latest refresh
+    pub const MAX_PROOF_VALIDITY: i64 = (EPOCHS_IN_YEAR * 3) / 2;
+
+    /// A number of epochs before proof expiration during which the proof can be extended
+    pub const PROOF_REFRESH_WINDOW: i64 =
+        MAX_PROOF_VALIDITY - (5*EPOCHS_IN_YEAR /* sector max life */ - MAX_PROOF_VALIDITY) / 3;
```

Comment:
This is a bit opaque, and I'm not sure we'd automatically want this to change if MAX_PROOF_VALIDITY changed. Just write in the number, and then explain in a comment how it relates to the others.